src/content-helper/common/css/variables.scss

```diff
@@ -46,4 +47,5 @@ $html-font-size: 16px; // Used in functions.scss.
 	--font-size--extra-large: 1.2rem;
 	--black: #000;
 	--sidebar-black: #1e1e1e;
+	--parsely-green: hsl(107, 42%, 46%);
```

Comment:
Great! Let's move this below line 18. This way, we'll exactly match the original stylesheet's order.